packages/manager/apps/dedicated/client/app/account/contacts/user-contacts.service.js

```diff
@@ -21,7 +19,28 @@ angular
     };
 
     self.noAvailableService = ['IP', 'VRACK'];
-    self.availableService = AVAILABLE_SERVICES;
+    self.availableService = [
```

Comment:
same here this has been replaced with constants

Reply:
Resolved. Has been refactored